src/config.py

```diff
@@ -51,6 +53,9 @@ class LogRotate:
 
         MAX_LOG_SIZE = "50M"
         MAX_ROTATIONS_TO_KEEP = 10
+        LOG_ROTATE_TEMPLATE = "templates/logrotate.j2"
```

Comment:
Isn't it it safer to have a relative path here?
```suggestion
        LOG_ROTATE_TEMPLATE = "./templates/logrotate.j2"
```